SeaBattle/Ship.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeaBattle
+{
+    public class BaseShip
+    {
+        public BaseShip (string Name, List<PointStruct> points) { }
+        private List<Deck> Decks { get; set; }
+        int Durability;
+        int MoveSpeed;
+        int MoveDirectionX;
+        int MoveDirectionY;
+
+    }
+
+    public class Deck
+    {
+        public PointStruct Location;
+
+        // If 0 = SANK, 1 = AFLOAT.
+        public int State;
```

Comment:
so this variable stores 2 states and you are using for this INT? why?

Reply:
Two states is bool true/false I think, but I am not sure I need states at all, since my ships will have HP. I will ask in Skype about it.

Reply:
of course, you have to. new type in the new file